mixer/adapter/dogstatsd/operatorconfig/attributes.yaml

```diff
@@ -0,0 +1,131 @@
+apiVersion: "config.istio.io/v1alpha2"
```

Comment:
why duplicate this locally?

Reply:
I'd seen it in another adapter and figured it was a test fixture of some sort, I'll clean it up